FashionConnect_api/routes/users.js

```diff
@@ -80,16 +80,14 @@ router.post('/users/profile', async (req, res) => {
       const user = await User.findOne({ where: { username:username } });
       console.log(user)
   
-      if (!user) {
```

Comment:
Any reason why you removed these lines? Looks like you would need them?

Reply:
I added it at first to check if the user s editing his/her accurate information. It must have been deleted by mistake.